html/playerview.html

```diff
@@ -14,6 +14,19 @@
 </head>
 
 <body id="playerview">
+
+    <div id="imgModal" class="modal" data-bind="visible: imageModalVisible, click: CloseImageModal()">
+        <span class="close">&times;</span>
```

Comment:
Use a fontawesome class here (take a look at the way other buttons in the app are implemented)